src/Microsoft.AspNet.Diagnostics/Views/ErrorPage.cshtml

```diff
@@ -176,10 +176,13 @@
                     <tbody>
                         @foreach (var kv in Model.Cookies.OrderBy(kv => kv.Key))
                         {
-                            <tr>
-                                <td>@kv.Key</td>
-                                <td>@kv.Value</td>
-                            </tr>
+                            foreach (var v in kv.Value)
```

Comment:
Similar to what we have in
https://github.com/aspnet/Diagnostics/blob/dev/src/Microsoft.AspNet.Diagnostics/Views/ErrorPage.cshtml#L147
and
https://github.com/aspnet/Diagnostics/blob/dev/src/Microsoft.AspNet.Diagnostics/Views/ErrorPage.cshtml#L203